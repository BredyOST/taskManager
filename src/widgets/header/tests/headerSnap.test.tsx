import {render} from "@testing-library/react";
import Header from "../header.tsx";


describe('header', () => {
    test('matches snapshot', () => {
        const {asFragment} = render(<Header />)
        expect(asFragment()).toMatchSnapshot()
    })
})