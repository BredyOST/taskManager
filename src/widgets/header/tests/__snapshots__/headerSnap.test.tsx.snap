// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-blHHSb kyxRVO"
  >
    <h3
      class="sc-gtLWhw vZxyv"
    >
      todos
    </h3>
  </div>
</DocumentFragment>
`;
